### ISSUE TITLE
Code fragments no longer work in markdown with highlightjs

### ISSUE THREAD

**Issue by nicojs** [hakimel/reveal.js#2981]
Code fragments annotated with `<!-- .element class="fragment" -->` no longer seems to work from reveal.js 4.1 when combining the markdown plugin with the highlight plugin.

For example (assuming [installed from npm](https://revealjs.com/installation/#installing-from-npm)).

```html
  <div class="reveal">
        <div class="slides">
            <section data-markdown>
                <textarea data-template>
                    ## Slide 1
                    A paragraph with some text and a [link](http://hakim.se).
                    
                    ```js
                    foo.bar();
                    ```

                    
                    
                    Code annotation

                    

                </textarea>
            </section>
        </div>
    </div>
    <script type="importmap">
        {
          "imports": {
            "reveal.js": "/node_modules/reveal.js/dist/reveal.esm.js",
            "reveal.js/plugin/markdown/markdown.esm.js": "/node_modules/reveal.js/plugin/markdown/markdown.esm.js",
            "reveal.js/plugin/highlight/highlight.esm.js": "/node_modules/reveal.js/plugin/highlight/highlight.esm.js"
          }
        }
        </script>
    <script type="module">
        import Reveal from 'reveal.js';
        import Markdown from 'reveal.js/plugin/markdown/markdown.esm.js';
        import Highlight from 'reveal.js/plugin/highlight/highlight.esm.js'

        let deck = new Reveal({
            plugins: [ Markdown, Highlight ]
        });
        deck.initialize();
    </script>
```

I would assume the code example to appear after a click or keyboard action, instead, it appears right away. Note: in reveal.js 4.0 this was still working.

When I inspect the DOM, I can see that the `pre` doesn't have a fragment class, instead, it has a `code-wrapper` class:

![image](https://user-images.githubusercontent.com/1828233/124554590-6b9d4000-de36-11eb-9d71-88cb54c65d4d.png)

**Comment by hakimel** [comment]
The fix for this has been merged in #2982

**Comment by nicojs** [comment]
Thanks for merging @hakimel ❤

When are you planning to release the fix?
